lib/rhc/ssh_helpers.rb

```diff
@@ -254,7 +254,7 @@ def save_snapshot(app, filename, for_deployment=false, ssh_executable=nil)
 
       snapshot_cmd = for_deployment ? 'gear archive-deployment' : 'snapshot'
       ssh_cmd = "#{ssh_executable} #{ssh_uri.user}@#{ssh_uri.host} '#{snapshot_cmd}' > #{filename}"
-      ssh_stderr = " 2>/dev/null"
+      ssh_stderr = debug? ? '' : ' 2>/dev/null'
```

Comment:
Nice catch. I'm remove the condition and keep this line how it is to keep things consistent with other methods.